### ISSUE TITLE
Amazing Feature !

### ISSUE THREAD

**Issue by terrypa** [dslul/openboard#414]
First of all, just installed it, love it! 
 
 TL:DR in one picture:
 Just swipe for all useful functionality from you normal keyboard
[https://i.imgur.com/bXE23W7.png](url)
 
 
 
 Longer version:
 I propose feature that was once present on "Swype" keyboard back in the early android [https://www.androidpolice.com/2018/02/19/swype-keyboard-now-discontinued/](url) and this  was awesome functionality, it gave user same possibility as on ordinary keyboard on PC,
 -no more rewriting from scratch after unintentional delete just swipe 123 to X = (CTRL+X)
 -no more pressing delete just 123 to Z =(CTRL+Z)
 -no need for unnecessary holding for selecting all message just swipe from 123 to A = (CTRL+A)
 -faster first and last line select
 - and so on
 
 They implemented back then, most of PC functionality, the same way u had implemented swipe switch from 
 CTR+A
 CTRL+C
 CTRL+V
 CTRL+Z
 CTRL+X
 HOME
 END
 and maybe some more PageUP, PageDn (I don't remember the rest but it dose open other possible shortcuts)
 
 Overall since i had experience it, I can't forget it, no other keyboard since then had it, a lot of tasks was so much faster i miss this sooo much, I'm convinced if any of you will experience it, you will not be able to go back.
 
 

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by PLAYERUNKNOWNHatsune01** [comment]
Is this work on Dvorak ???

**Comment by Yur7y** [comment]
Nintype (Keyboard 69) on the playstore has this feature as well. https://play.google.com/store/apps/details?id=com.jormy.sixtynine

I came here to ask for this feature as well. Keyboard 69 does it a little differently though, you execute the commands by dragging down from one of the letters to the space key. (Aka drag down from V to the space bar to paste your text, drag down from C to space to copy your text) and it’s customizable so you can choose which keys you want to do which actions. 

OpenBoard already has the ability to swipe the backspace to delete whole words, would be really awesome if it had more gestures added for other tasks!
